site/fi_FI.all.json

```diff
@@ -446,156 +446,153 @@
   "donate_button_text": {
     "other": "Lahjoittaa"
   },
-  "wcag_skip_links_header": { "other": "Accessibility Links" },
-  "wcag_skip_links_content": { "other": "Skip to Content" },
 
+  "wcag_skip_links_header": { "other": "Esteettömyyslinkit" },
+  "wcag_skip_links_content": { "other": "Siirry sisältöön" },
 
-  "wcag_homepage_h1": { "other": "Homepage" },
-  "wcag_carousel_h2": { "other": "Carousel" },
-  "wcag_collections_h2": { "other": "Collections" },
+  "wcag_homepage_h1": { "other": "Kotisivu" },
+  "wcag_carousel_h2": { "other": "Karuselli" },
+  "wcag_collections_h2": { "other": "kokoelma" },
 
-  "wcag_aria_label_footer": { "other": "Footer" },
-  "wcag_aria_label_nav_main": { "other": "Main" },
+  "wcag_aria_label_footer": { "other": "Alatunniste" },
+  "wcag_aria_label_nav_main": { "other": "Tärkein" },
   "wcag_aria_label_nav_sub": { "other": "Sub" },
-  "wcag_aria_label_toggle_nav": { "other": "Toggle Navigation" },
-  "wcag_aria_label_bundle_items": { "other": "Bundle Items" },
-  "wcag_aria_label_carousel": { "other": "Carousel" },
-  "wcag_aria_label_page_collection": { "other": "Page Collection" },
-  "wcag_aria_label_collection_list": { "other": "Collection List" },
-  "wcag_aria_label_collection_slider": { "other": "Collection Slider" },
-  "wcag_aria_label_wishlist": { "other": "Wishlist" },
-  "wcag_aria_label_facebook": { "other": "Share on Facebook" },
-  "wcag_aria_label_twitter": { "other": "Share on Twitter" },
-  "wcag_aria_label_linkedin": { "other": "Share on LinkedIn" },
-  "wcag_aria_label_letterboxd": { "other": "View on Letterboxd" },
+  "wcag_aria_label_toggle_nav": { "other": "Navigointi päälle/pois" },
+  "wcag_aria_label_bundle_items": { "other": "Niputa kohteita" },
+  "wcag_aria_label_carousel": { "other": "Karuselli" },
+  "wcag_aria_label_page_collection": { "other": "Sivujen kokoelma" },
+  "wcag_aria_label_collection_list": { "other": "Kokoelmaluettelo" },
+  "wcag_aria_label_collection_slider": { "other": "Kokoelman liukusäädin" },
+  "wcag_aria_label_wishlist": { "other": "Toivelista" },
+  "wcag_aria_label_facebook": { "other": "Jaa Facebookissa" },
+  "wcag_aria_label_twitter": { "other": "Jaa Twitterissä" },
+  "wcag_aria_label_linkedin": { "other": "Jaa LinkedInissä" },
```

Comment:
Ah, this is what I did too with the company names.

Reply:
shouldn't the company name be in their original form?

Reply:
I don't really know myself - I figured if we are using Google Translate as our bare minimum I was just going to trust it...

Reply:
its kind of hard to answer this one lol 
I think both yes and no, the company name by itself translates one to one, but adding the action such as "jaa/share on" will add the 'issa' on the end, I suspect there are some Finnish noun/ grammar rules coming into play, the Wiki for twitter seems to use both Twitter/Twitterissa, depending on context.

Reply:
Also interesting the grammar rule has the `a` vs `ä`.

Reply:
I'm tempted to go with the `issa` suffix as the `social_media_buttons_X` key values also include them, they were translated before these changes, so I suspect translated by a Finnish client?